lilypad/developer-resources/create-lilypad-module/building-your-module.md

```diff
@@ -0,0 +1,349 @@
+---
+description: Creating a Lilypad job module
+---
+
+# Creating Your Module
+
+In this guide, we'll be creating a basic sentiment analysis module using [`distilbert/distilbert-base-uncased-finetuned-sst-2-english`](https://huggingface.co/distilbert/distilbert-base-uncased-finetuned-sst-2-english) (which I will be referring to as Distilbert from now on). We will be referring back to the Hugging Face page throughout this guide, so it's best to keep it open and accessible.
```

Comment:
Remove first person

"which I will be referring to as Distilbert from now on"
to
"which will be referred to as Distilbert from now on"